metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "treydock-kdump",
-  "version": "0.0.1",
+  "version": "0.1.0",
```

Comment:
Because dropping Puppet 3 should probably go ahead and make this `1.0.0`.  Noticed voxpupuli tends to bump major version when dropping support for versions of Puppet and I've been following same pattern with other modules.

Reply:
Technically, pre 1.0.0, semver says you can do what you like.

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

Reply:
Good point, in that case 0.1.0 is fine.

Reply:
I left the version 0.1.0.  I will leave it up to you guys to set  the version as you see fit.  I think everyone interprets semver a little different.